helix-term/src/application.rs

```diff
@@ -52,30 +52,57 @@ pub struct Application {
 impl Application {
     pub fn new(args: Args, mut config: Config) -> Result<Self, Error> {
         use helix_view::editor::Action;
-        let mut compositor = Compositor::new()?;
-        let size = compositor.size();
 
-        let conf_dir = helix_core::config_dir();
+        let config_dir = helix_core::config_dir();
+        let local_config_dir = helix_core::local_config_dir().unwrap();
```

Comment:
Note: Should instead unwrap in the `local_config_dir()` function.

Reply:
There should be no unwraps, it's perfectly valid not to find a root. You could use cwd in that case or just do nothing.